ray-operator/controllers/ray/common/pod.go

```diff
@@ -417,6 +418,16 @@ func concatenateContainerCommand(nodeType rayiov1alpha1.RayNodeType, rayStartPar
 		}
 	}
 
+	if _, ok := rayStartParams["dashboard-agent-listen-port"]; !ok {
+		rayStartParams["dashboard-agent-listen-port"] = fmt.Sprint(DefaultDashboardAgentListenPort)
+		for _, port := range ports {
+			if port.Name == "dashboard-agent" {
+				rayStartParams["dashboard-agent-listen-port"] = fmt.Sprint(port.ContainerPort)
```

Comment:
Is dashboard-agent-listen-port a new arg added to Ray nightly?
Is this going to crash earlier versions of Ray?

Reply:
If so, one approach would be to add a toggle to turn the new behavior on/off.
If the toggle is on, it could potentially also auto-configure the port, so that the user has to do less.

Reply:
It already exists

Reply:
Ah, yep it's been there since Ray 1.6.0 -- early August 2021.
Should be fine -- I don't think there's need to keep compatibility that far back.